change-notes/1.19/analysis-java.md

```diff
@@ -26,6 +26,11 @@ to run queries and explore the data flow in results.
 
 ## Changes to QL libraries
 
+* The class `ControlFlowNode` (and by extension `BasicBlock`) has until now
+  been directly equatable to `Expr` and `Stmt`.  Exploiting these equalities,
+  for example by using casts, is now deprecated, and the conversions
+  `Expr.getControlFlowNode()` and `Stmt.getControlFlowNode()` should be used
+  instead.
```

Comment:
Is it worth including an example of before and after, or is this obvious to users?

Reply:
This is mostly a warning about a breaking change we've agreed to put in 1.20, which we've convinced ourselves hopefully won't affect too many.  The easiest is to just ignore the 1.19 deprecation and fix any compile errors when upgrading to 1.20, so we'll add more information in the 1.20 change notes.

Reply:
Okay - makes sense. I've raised https://jira.semmle.com/browse/SD-2920 for the changes required to publish this change for 1.19.